R/extract.R

```diff
@@ -2273,6 +2273,101 @@ setMethod("extract", signature = className("gamlss", "gamlss"),
           definition = extract.gamlss)
 
 
+# -- extract.gamlssZadj (gamlss.inf) -------------------------------------------
+
+#' @noRd
+extract.gamlssZadj <- function(model, type = c("qr", "vcov"), robust = FALSE,
+                               include.nobs = TRUE, include.gaic = TRUE, ...) {
+
+  type <- match.arg(type)
+  # VCOV extraction; create coefficient block
+  if (type == "vcov") {
+    covmat <- suppressWarnings(stats::vcov(model, type = "all", robust = robust,
```

Comment:
When `robust = TRUE` and `type = "vcov"`, the `vcov` call produces an error, at least with the default help page example of `gamlssZadj`. Can you check and fix? See also the last line in the unit test to produce the error.